tensorcircuit/mpscircuit.py

```diff
@@ -396,21 +661,25 @@ def wavefunction(self, form: str = "default") -> Tensor:
 
     state = wavefunction
 
-    # TODO(@refraction-ray): mps form quvector
-
     def copy_without_tensor(self) -> "MPSCircuit":
         """
         Copy the current MPS without the tensors.
 
         :return: The constructed MPS
         :rtype: MPSCircuit
         """
+        from copy import copy
```

Comment:
why import copy inside the method?

Reply:
and what is the use of this new MPSCircuit method

Reply:
just to copy the attributes to avoid unwanted change of the original object. This method is actually not designed for user to use outside.

Reply:
again, when import copy here instead of file level?

Reply:
done